File: dataset/output/OPENAI_gpt-4o/Task44_OPENAI_gpt-4o.java
import java.util.Scanner;

public class MultiLanguageProgram {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an integer: ");
        if (scanner.hasNextInt()) {
            int userInput = scanner.nextInt();
            int result = performOperation(userInput);
            System.out.println("The result from Java operation is: " + result);
        } else {
            System.out.println("Invalid input.");
        }
    }

    private static int performOperation(int input) {
        return input * 2;
    }
}

File: dataset/output-bk/OPENAI_gpt-4o/Task44_OPENAI_gpt-4o.cpp
#include <iostream>

int performOperation(int input);

int main() {
    int userInput;
    std::cout << "Enter an integer: ";
    if (std::cin >> userInput) {
        int result = performOperation(userInput);
        std::cout << "The result from C++ operation is: " << result << std::endl;
    } else {
        std::cout << "Invalid input." << std::endl;
    }
    return 0;
}

int performOperation(int input) {
    return input * 2;
}